Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs

```diff
@@ -12,7 +12,7 @@
 namespace FluentAssertions.Specs.Formatting;
 
 [Collection("FormatterSpecs")]
-public sealed class FormatterSpecs : IDisposable
+public class FormatterSpecs : IDisposable
```

Comment:
```suggestion
public sealed class FormatterSpecs : IDisposable
```

Reply:
Why do we care?

Reply:
I recall I/we applied it because an analyzer pointed out that Dispose should be virtual in a non-sealed class.

Reply:
Ah, I disabled those for tests. I see no need to spend time on those details for test projects.